docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -923,6 +942,29 @@ logic. In this example, both the event's `destination.ip` and
 <4> Sibling `entries` are evaluated using `or` logic. An alert is generated when
 at least one `entries` object evaluates to `true`.
 
+*Example 6*
+
+New terms rule that creates alerts when the Windows `rundll32.exe` process makes
+unusual network connections:
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules
+{
+  "risk_score": 21,
+  "description": "Detects a user associated with a new IP address",
+  "name": "New User IP Detected",
+  "severity": "medium",
```

Comment:
Should `severity` correspond to the `risk_score` value, i.e. should it be `low` as `risk_score` is `21` ?